src/components/Map/Map.js

```diff
@@ -168,15 +173,30 @@ export default class Map extends React.Component<Props, State> {
       );
     }
 
-    const zoom = overrideZoom || fallbackZoom;
-
-    const placeOrEquipment = state.placeOrEquipment;
-
+    let zoom = overrideZoom || fallbackZoom;
     let center = [0, 0];
     let bounds = null;
     let zoomedToFeatureId = state.zoomedToFeatureId;
+    let centerDefined = false;
```

Comment:
I have to admit I don't fully understand the logic behind this flag… could you add a comment what it does? Maybe renaming is even enough :)

Reply:
Will do!